core/src/main/java/com/graphhopper/util/InstructionList.java

```diff
@@ -334,4 +424,15 @@ public Instruction find(double lat, double lon, double maxDistance) {
         return get(foundInstruction);
     }
 
+    public void setPoints(PointList points) {
+        this.points = points;
+    }
+
+    public PointList getPoints(Instruction instruction) {
```

Comment:
Maybe a different name or some javadocs are important to see the difference between this method and getPoints()?

Reply:
Yes certainly, it's still pretty raw and I am trying to get everything working, as it might involve bigger changes at some points in the code (not 100% sure yet).